youtube-client/src/components/Card.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import Card from '@mui/material/Card';
+import CardHeader from '@mui/material/CardHeader';
+import CardMedia from '@mui/material/CardMedia';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import PersonIcon from '@mui/icons-material/Person';
+import DateRangeIcon from '@mui/icons-material/DateRange';
+import VisibilityIcon from '@mui/icons-material/Visibility';
+import styled from 'styled-components';
+import { Video } from '../interfaces';
+
+const InfoContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: start;
+  padding-left: 16px;
+`;
+
+const InfoText = styled.div`
+  display: block;
+  margin: 0;
+  margin-left: 15px;
+`;
+
+type VideoCardProps = {
+  video: Video;
+};
+
+export default function VideoCard(props: VideoCardProps): JSX.Element {
+  const { video } = props;
+
+  return (
+    <Card sx={{ width: 250, height: 400, margin: '0 auto' }}>
+      <CardHeader title={video.snippet.title} sx={{ paddingBottom: 0, marginBottom: '16px', overflow: 'hidden', whiteSpace: 'nowrap', textOverflow: 'ellipsis', width: '250px' }} />
```

Comment:
text overflow ellipsis applied to incorrect element, 
it looks like this:
![image](https://user-images.githubusercontent.com/9293008/158814077-1376f65f-7bd0-4263-bbc2-3433ad65d81f.png)
after the small adjustment it should look like this:
![image](https://user-images.githubusercontent.com/9293008/158814197-88a805c1-73c7-4f0d-87b2-30c5a3e3316e.png)